### PR TITLE
Java11

### PR DESCRIPTION
**Pull request by rushalias**
Following the same patten used in http server, added a flag which will enable us to disable list of protocols.   For now this flag `-Dot.httpserver.ssl-excluded-protocols=TLSv1.3` has been added to otpl-common-config[Java11 branch]/jvm.properties file.

### CODE REVIEW

File: otj-server-core/src/main/java/com/opentable/server/EmbeddedJettyBase.java [hunk 1]
```diff
@@ -263,6 +267,12 @@ private ConnectorInfo createConnector(Server server, String name, IntSupplier po
         final HttpConnectionFactory http = new HttpConnectionFactory(httpConfig);
 
         if (ssl != null) {
+            if (!CollectionUtils.isEmpty(excludedProtocols)) {
+                LOG.warn("Excluding following protocols:");
```
Comment (adamzr) [inline otj-server-core/src/main/java/com/opentable/server/EmbeddedJettyBase.java]: Probably should log at the info level here too

Comment (mikebell90) [reply otj-server-core/src/main/java/com/opentable/server/EmbeddedJettyBase.java]: Warn is higher than info ain't it

Comment (rushalias) [reply otj-server-core/src/main/java/com/opentable/server/EmbeddedJettyBase.java]: sticking with warn, same level of logging is used in otj server to note the fact its' running with old ciphers.

File: pom.xml [hunk 2]
```diff
@@ -40,32 +40,37 @@
...
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.apache.maven.shared</groupId>
+                        <artifactId>maven-dependency-analyzer</artifactId>
+                        <version>1.11.1</version>
+                        <exclusions>
+                            <exclusion>
```
Comment (adamzr) [inline pom.xml]: Why is this needed here?

Comment (mikebell90) [reply pom.xml]: It isn't and can be removed in future.
